mlaut/benchmarking/evaluation.py

```diff
@@ -0,0 +1,525 @@
+__author__ = ["Viktor Kazakov", "Markus Löning", "Aaron Bostrom"]
+__all__ = ["Evaluator"]
+
+import itertools
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import scikit_posthocs as sp
+from scipy import stats
+from scipy.stats import ranksums
+from scipy.stats import ttest_ind
+
+from mlaut.benchmarking.base import BaseResults
+from mlaut.shared.exceptions import NotEvaluatedError
+
+plt.style.use("seaborn-ticks")
+
+
+class Evaluator:
```

Comment:
We should make clear that this evaluator is for the "multiple datasets and one or more estimators" case, it'd be good to have a separate evaluator for the "single dataset and one or more estimators" case. I came across a [question](https://stats.stackexchange.com/questions/365370/analogue-of-critical-difference-cd-diagram-for-the-comparison-over-single-data) on stack exchange for that case the other day.